test/wrapper.test.ts

```diff
@@ -155,8 +155,13 @@ describe("Tests Native Wrapper", () => {
 
   describe("deviceContexts", () => {
     test("fetches the contexts from native", async () => {
-      await expect(NATIVE.deviceContexts()).resolves.toMatchObject({});
+      await expect(NATIVE.deviceContexts()).resolves.toMatchObject({ someContext: 0});
     });
+    test("returns empty object if not on ios", async () => {
```

Comment:
The test descriptions seem inverted

Reply:
Hmm, it seems correct to me, but I think I'll make it more clear.